packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx

```diff
@@ -16,137 +20,164 @@ const VisualizationCardOptions: FC = () => {
     const disabled = isLoading || plotData.length <= 0;
     const cartesianType = cartesianConfig.dirtyChartType;
     const cartesianFlipAxis = cartesianConfig.dirtyLayout?.flipAxes;
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+
+    const selectChartType = () => {
+        if (chartType === ChartType.CARTESIAN) {
```

Comment:
Since **chartType** and **cartesianType** can't be undefined we can improve our typing by using a switch + type never in the default case.
By having that combination we make sure once there is a new chart/cartesian type we don't forget to add a new switch case in this component since it will cause a linting error. 

E.g from another component
```
switch (type) {
        case DashboardTileTypes.SAVED_CHART:
            return ChartTileForm;
        case DashboardTileTypes.MARKDOWN:
            return MarkdownTileForm;
        case DashboardTileTypes.LOOM:
            return LoomTileForm;
        default: {
            const never: never = type;
            throw new Error('Tile type not supported');
        }
    }
```